packages/core/js-client/src/connection/RelayConnection.ts

```diff
@@ -80,6 +80,19 @@ export interface RelayConnectionConfig {
   maxOutboundStreams: number;
 }
 
+type DenyCondition = (ma: Multiaddr) => boolean;
+
+const DOCKER_NOX_DENY_CONDITION: DenyCondition = (ma) => {
+  const [routingProtocol] = ma.stringTuples();
+  const host = routingProtocol?.[1];
+
+  return (
+    host === undefined || host.startsWith("nox-") || host.startsWith("10.50.10")
```

Comment:
I am understanding correctly that when this is true - then dialing this multiaddr is forbidden?
why specifically `nox-` and `10.50.10`. Why be nice to leave the comment about that

Reply:
there should be a more general way to handle that

Reply:
what if non-available addresses returned in real life, not in tests?